### PR TITLE
Add ability to only run or exclude specific test cases within a test …

### PR DESCRIPTION
**Pull request by dasormeter**
…suite

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags, if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala [hunk 1]
```diff
@@ -94,6 +94,7 @@ object LedgerApiTestTool {
 
     val testsToRun = Tests.all.filterKeys(included -- config.excluded)
 
+
```
Comment (SamirTalwar) [inline ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala]: I feel like you probably meant to do more here. As it is you don't seem to use those new fields at all.

Comment (SamirTalwar) [reply ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala]: Oh, sorry, didn't see this was a draft. Carry on!

**Comment by dasormeter** [comment]
needs some love still

**Comment by garyverhaegen-da** [comment]
I believe this is superseded by #6458. @dasormeter please reopen if I'm mistaken.

**Pull request closed without merging**